gradle/libs.versions.toml

```diff
@@ -1,27 +1,24 @@
 [versions]
 
-gradlePlugin-kotlin = "1.9.22"
+gradlePlugin-kotlin = "2.0.0-Beta4"
 # See: https://kotlinlang.org/docs/gradle-configure-project.html#apply-the-plugin
-gradlePlugin-android = "4.2.2"
-gradlePlugin-android-dokkatoo = "8.0.2"
+gradlePlugin-android = "7.1.3"
 gradlePlugin-dokka = "1.9.20"
-gradlePlugin-gradle-customUserData = "1.12"
-gradlePlugin-gradle-enterprise = "3.14.1"
 
 kotlinx-coroutines = "1.7.3"
 kotlinx-collections-immutable = "0.3.6"
 kotlinx-serialization = "1.6.0"
 kotlinx-bcv = "0.13.2"
 
 ## Analysis
-kotlin-compiler = "1.9.22"
+kotlin-compiler = "2.0.0-Beta4"
 kotlin-compiler-k2 = "2.0.0-dev-8561"
 
 # MUST match the version of the intellij platform used in the kotlin compiler,
 # otherwise this will lead to different versions of psi API and implementations
 # on the classpath and will fail with hard to debug problems in runtime.
 # See: https://github.com/JetBrains/kotlin/blob/e6633d3d9214402fcf3585ae8c24213a4761cc8b/gradle/versions.properties#L1
-intellij-platform = "213.7172.25"
+intellij-platform = "213.7172.53"
```

Comment:
IntelliJ platform was bumped in 2.0.0-Beta4

https://github.com/JetBrains/kotlin/blob/9554232c35d628b5948c23c1b96dde89bcab074a/gradle/versions.properties#L1